torch/csrc/distributed/rpc/metrics/RpcMetricsHandler.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <string>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+// All metrics are prefixed with the following  key.
+const std::string kRPCMetricsKeyPrefix = "torch.distributed.rpc.";
+// Thread-safe API for logging time-series metrics for RPC-based model parallel
+// training. This class provides thread safety so that metrics can be logged
+// from multiple threads without the user needing to coordinate serialization.
+// The class provides two main functions - accumulateMetric which can be used
+// for statistical aggregations and setMetric for setting/modifying single
+// values.
+class RpcMetricsHandler {
+ public:
+  // Log value corresponding to metric name given by name to the time-series
+  // metric stream.
```

Comment:
This seems like implementation detail. The doc should say something like this "Accumulates the metric value specified for purposes of computing average/percentiles over time".